[ML] Set default value of 30 days for model prune window (#81377)
elastic/elasticsearch: docs/reference/ml/ml-shared.asciidoc

Affects the pruning of models that have not been updated for the given time
duration. The value must be set to a multiple of the `bucket_span`. If set too
low, important information may be removed from the model. Typically, set to

> `30d` or longer. If not set a default value of 30 days is used.

Reviewer: ```suggestion
`30d` or longer. If not set, model pruning only occurs if the model memory
 status reaches the soft limit or the hard limit. For jobs created in 8.1 and
 above, a default value of 30 days is set on job creation if no other value is
 set.
```

duration. The value must be set to a multiple of the `bucket_span`. If set too
low, important information may be removed from the model. Typically, set to
`30d` or longer. If not set, model pruning only occurs if the model memory
status reaches the soft limit or the hard limit. For jobs created in 8.1 and

> later, the default value is the greater of `30d` or 20 * `bucket_span`.

Reviewer: @lcawl do you have a view on whether it should be "20 times `bucket_span`" instead of "20 * `bucket_span`"?

Reviewer 2: I'd vote for the former for clarity.

duration. The value must be set to a multiple of the `bucket_span`. If set too
low, important information may be removed from the model. Typically, set to
`30d` or longer. If not set, model pruning only occurs if the model memory
status reaches the soft limit or the hard limit. For jobs created in 8.1 and

> above, a default value of 30 days is set on job creation if no other value is

Reviewer 2: I think this can be simplified to:
```suggestion
later, the default value is `30d`.
```

> set.

Reviewer 2: If you accept my other suggestion, this one is also required:
```suggestion

```

>  set.

Reviewer: It looks like this is indented by 1 space, probably because of the way my earlier suggestion went in directly via GitHub.
